src/zcbenchmarks.cpp

```diff
@@ -116,10 +117,24 @@ double benchmark_solve_equihash()
                                     nonce.begin(),
                                     nonce.size());
 
-    timer_start();
+    if (time)
+        timer_start();
```

Comment:
Ah I see now. Note that the threaded benchmark includes the time to serialise and hash the block header+nonce, while the unthreaded benchmark doesn't. I don't think this is a big problem, but probably it's more realistic as well as more consistent to include this always.